src/lib/leagues/mediumTasks.ts

```diff
@@ -0,0 +1,903 @@
+		}
+	},
+	{
+		id: 1100,
+		name: 'Reach 15% CL completion',
+		has: async ({ clPercent }) => {
+			return clPercent >= 15;
+		}
+	},
+	{
+		id: 1101,
+		name: 'Construct 10,000 objects',
+		has: async ({ conStats }) => {
+			return sumArr(conStats.items().map(i => i[1])) >= 10_000;
+		}
+	},
+	{
+		id: 1102,
+		name: 'Chop 5000 of any logs',
+		has: async ({ woodcuttingStats }) => {
+			return sumArr(woodcuttingStats.items().map(i => i[1])) >= 5000;
+		}
+	},
+	{
+		id: 1103,
+		name: 'Alch 1000 of any item',
+		has: async ({ alchingStats }) => {
+			return sumArr(alchingStats.items().map(i => i[1])) >= 1000;
+		}
+	},
+	{
+		id: 1104,
+		name: 'Clean 500 herbs',
+		has: async ({ herbloreStats }) => {
+			return sumArr(herbloreStats.herbs.items().map(i => i[1])) >= 500;
+		}
+	},
+	{
+		id: 1105,
+		name: 'Mix 500 unf potions',
+		has: async ({ herbloreStats }) => {
+			return sumArr(herbloreStats.unfPots.items().map(i => i[1])) >= 500;
+		}
+	},
+	{
+		id: 1106,
+		name: 'Mix 500 potions',
+		has: async ({ herbloreStats }) => {
+			return sumArr(herbloreStats.pots.items().map(i => i[1])) >= 500;
+		}
+	},
+	{
+		id: 1107,
+		name: 'Mine 1000 ores',
+		has: async ({ miningStats }) => {
+			return sumArr(miningStats.items().map(i => i[1])) >= 1000;
+		}
+	},
+	{
+		id: 1108,
+		name: 'Burn 500 logs',
+		has: async ({ firemakingStats }) => {
+			return sumArr(firemakingStats.items().map(i => i[1])) >= 500;
+		}
+	},
+	{
+		id: 1109,
+		name: 'Smith items from 500 Iron bars',
+		has: async ({ smithingSuppliesUsed }) => {
+			return smithingSuppliesUsed.amount('Iron bar') >= 500;
+		}
+	},
+	{
+		id: 1110,
+		name: 'Smith items from 1000 Bronze bars',
+		has: async ({ smithingSuppliesUsed }) => {
+			return smithingSuppliesUsed.amount('Bronze bar') >= 1000;
+		}
+	},
+	{
+		id: 1111,
+		name: 'Cast 500 spells',
+		has: async ({ spellCastingStats }) => {
+			return sumArr(spellCastingStats.map(i => i.qty)) >= 500;
+		}
+	},
+	{
+		id: 1112,
+		name: 'Collect 500 Blue dragon scales',
+		has: async ({ collectingStats }) => {
+			return collectingStats.amount('Blue dragon scale') >= 500;
+		}
+	},
+	{
+		id: 1113,
+		name: 'Collect 500 Mort myre fungus',
+		has: async ({ collectingStats }) => {
+			return collectingStats.amount('Mort myre fungus') >= 500;
+		}
+	},
+	{
+		id: 1114,
+		name: "Collect 500 Red spider's eggs",
+		has: async ({ collectingStats }) => {
+			return collectingStats.amount("Red spiders' eggs") >= 500;
+		}
+	},
+	{
+		id: 1115,
+		name: 'Smith items from 2500 Steel bars',
+		has: async ({ smithingSuppliesUsed }) => {
+			return smithingSuppliesUsed.amount('Steel bar') >= 2500;
+		}
+	},
+	{
+		id: 1116,
+		name: 'Create every godsword',
+		has: async ({ cl }) => {
+			return resolveItems([
+				'Armadyl godsword',
+				'Bandos godsword',
+				'Saradomin godsword',
+				'Zamorak godsword',
+				'Ancient godsword'
+			]).every(gs => cl.has(gs));
+		}
+	},
+	{
+		id: 1117,
+		name: 'Finish the Treebeard CL',
```

Comment:
I might rank this a little higher because you have to get all the different Tangleroot variants, so maybe Elite / Hard. But whatevers :D